main.go

```diff
@@ -31,17 +31,18 @@ const (
 
 func main() {
 	cnfg := config.DefaultConfig()
-	rootCmd, globalFlagValues := rootcmd.New(&cnfg.Hazelcast)
+	rootCmd, persistentRootFlags := rootcmd.New(&cnfg.Hazelcast)
 	programArgs := os.Args[1:]
 	// update config before running root command to make sure flags are processed
-	err := updateConfigWithFlags(rootCmd, cnfg, programArgs, globalFlagValues)
+	err := updateConfigWithFlags(rootCmd, cnfg, programArgs, persistentRootFlags)
```

Comment:
Why were these changes necessary?

Reply:
Switched back to previous form, [7a991c2](https://github.com/hazelcast/hazelcast-commandline-client/pull/67/commits/7a991c20b78270f67c8e763f4727f6ef1e3a3463).